### DISCUSSION TITLE
UI for Sotopia to collect human-AI data

### DISCUSSION THREAD

**Discussion by XuhuiZhou** [sotopia-lab/sotopia#39] [Ideas]
I am tagging people here:
@Jasonqi146 @lwaekfjlk Can you describe a bit what you are working on?

@Algoroxyolo feel free to sync with them!

**Comment by Jasonqi146** [comment]
Hi everyone, @lwaekfjlk and I am starting from building a AI/human conversation page with Gradio. It looks like the [WildBench](https://huggingface.co/spaces/allenai/WildBench) is more comprehensive page with different parts, and what we are working on now would be the evaluation tab in particular.

**Reply by XuhuiZhou** [reply]
Thanks @Jasonqi146!
@Algoroxyolo Feel free to on the `leaderboard` tab :)

**Comment by lwaekfjlk** [comment]
yep, our target is allowing the user to chat with multiple model agents.
